lib/app.js

```diff
@@ -0,0 +1,69 @@
+const HOOK_STAGE_PRE_INSTALL = 'preInstall'
+const HOOK_STAGE_POST_INSTALL = 'postInstall'
+
+const isObject = o => typeof o === 'object' && o !== null && !Array.isArray(o)
+
+const runHandler = handler => {
```

Comment:
Any reason not to use `async` syntax?

Reply:
We don't use `async` in `lib/` and it means we don't have to package `regenerator-runtime`. I don't feel like doing it just to use `async` in this one file. Converting the whole project would be a different story ;)

Reply:
Yep, makes sense.